linera-service/tests/linera_net_tests.rs

```diff
@@ -914,7 +904,18 @@ async fn test_wasm_end_to_end_same_wallet_fungible(
     );
 
     // Needed synchronization though removing it does not get error in 100% of cases.
-    assert_eq!(node_service.process_inbox(&chain1).await?.len(), 1);
+    let expected_processed_certificates = if example_name == "native-fungible" {
+        // native-fungible needs to do transfers on instantiation to properly set
+        // the initial balances of the accounts. So we'll have a `Credit` message being
+        // processed here.
+        1
+    } else {
+        0
+    };
```

Comment:
(We could also just not make the `process_inbox` call at all unless `example_name == "native_fungible"`.)